### ISSUE TITLE
No obvious correlation to official ERCs

### ISSUE THREAD

**Issue by aspiers** [0xacme/ERC404#2]
Hi there,

I checked the following:

- https://github.com/ethereum/ERCs
- https://github.com/ethereum/eips
- https://eips.ethereum.org/

and I couldn't see any mention of this ERC on any of them, so I'm confused how this ended up as ERC-404.  Am I missing something?

**Comment by Moeno614** [comment]
I guess thats why it's erc-404, like not found since it doesn't really exsist as a erc.
hats down to 0xacme for doing all that in the underground

**Comment by lightclient** [comment]
Yeah that isn't how ERCs work. You're confusing the community by not just following the pattern followed by thousands of other developers and standards.

**Comment by albchamo** [comment]
ERCs have a process. Why call it that way at all? It's not even proposed for comments.

**Comment by TimDaub** [comment]
I recommend the ERC committee to file a defensive trademark to not having to resort to impotent moral claims. This person here is as much authorized to name their project ERC404 as the ERC volunteers because there‘s simply no circumstance to derive any meaningful authority from. And I say this having submitted proposals to status:final in the ERC process. If the ERC committee pretends it has the authority to assign numbers globally, but it actually doesn‘t have the power to unassign foreign assignments, it has no global authority at all over the name.

**Comment by aspiers** [comment]
I'm sorry but that misses the point. This is nothing to do with trademarks or authorities, and no one claimed that this infringed any trademark or other legal construct.  It's also not about assigning numbers.

It's about taking a term which is widely understood across the whole community to mean a specific, highly structured process, and then using that term to refer to something which completely ignores that process.

This project could have been called XYZ-404, or PQR-404, or a million other things, but the author chose to call it ERC-404 instead.  Was that a coincidence?  Of course not.  Whether or not the intention was to deliberately mislead people into thinking it was a real ERC, that was the effect achieved.  Now there are numerous crypto media outlets all referring to ERC-404 as a standard, when it's not yet even a proposal for a standard.  This gives a dangerous veneer of credibility to something which hasn't earned it yet.

I'm all for innovation, and of course it's fine for anyone to publish stuff no matter how polished it is.  Just please don't do it in a misleading way.

**Comment by TimDaub** [comment]
If there‘s no trademark then the only way to really deal with this are the following options

1. Stop complaining and accept that the ERC community has no monopoly on the name ERC-XXX
2. File for a defensive trademark and threaten the next person that does that
3. Come up with a better permissionless identifier scheme than what currently exists where the ERC volunteers have the authority to assign numbers as it clearly leads to infrigement

Note that I didn‘t list „complaining“ as an option as I don‘t think it can fundamentally lead to much.

**Comment by aspiers** [comment]
There are more positive and constructive options available than the ones you list, e.g. 

4. The author does the right thing and either submits the proposal through the ERC process or removes the misleading ERC label.  In either case, the problem is easily solved.
5. If 4. doesn't happen, the community and crypto journalists highlight the misleading nature of the label so that people don't invest their wealth into projects based on the misunderstanding that this new approach to smart contracts has the same level of endorsement and potential as ERC-20 or ERC-721 or other genuine standards.  For example, I already persuaded Bankless to amend [their article on this project](https://www.bankless.com/erc-404-pandora) to clarify the risks associated with this misleading label:

![image](https://github.com/0xacme/ERC404/assets/100738/13fd4a31-91f5-486e-a363-92b6410dab92)

This proves that constructive action can have a positive effect:

![image](https://github.com/0xacme/ERC404/assets/100738/ab8b3216-2f2e-4b70-b1e1-f2e0ef35c8ea)

**Comment by 0xacme** [comment]
> There are more positive and constructive options available than the ones you list, e.g.
> 
> 4. The author does the right thing and either submits the proposal through the ERC process or removes the misleading ERC label.  In either case, the problem is easily solved.
> 5. If 4. doesn't happen, the community and crypto journalists highlight the misleading nature of the label so that people don't invest their wealth into projects based on the misunderstanding that this new approach to smart contracts has the same level of endorsement and potential as ERC-20 or ERC-721 or other genuine standards.  For example, I already persuaded Bankless to amend [their article on this project](https://www.bankless.com/erc-404-pandora) to clarify the risks associated with this misleading label:
> 
> ![image](https://private-user-images.githubusercontent.com/100738/303861089-13fd4a31-91f5-486e-a363-92b6410dab92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc1OTQ2NDEsIm5iZiI6MTcwNzU5NDM0MSwicGF0aCI6Ii8xMDA3MzgvMzAzODYxMDg5LTEzZmQ0YTMxLTkxZjUtNDg2ZS1hMzYzLTkyYjY0MTBkYWI5Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxMFQxOTQ1NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYWIzM2VlMjRjZWI2ZTExMTlhN2ViNzUxM2QyN2Q4NDE2NmU4NzM1YTE3NjBkNWEyMDhiMjg0YWU0ZTBjZDZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.du3zoww61-ekasd2M-UWpqYp1aGQpuCUZnGZYMKKwog)
> 
> This proves that constructive action can have a positive effect:
> 
> ![image](https://private-user-images.githubusercontent.com/100738/303860854-ab8b3216-2f2e-4b70-b1e1-f2e0ef35c8ea.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc1OTQ2NDEsIm5iZiI6MTcwNzU5NDM0MSwicGF0aCI6Ii8xMDA3MzgvMzAzODYwODU0LWFiOGIzMjE2LTJmMmUtNGI3MC1iMWUxLWYyZTBlZjM1YzhlYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxMFQxOTQ1NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMzc0NmMwNjM3YWNmZGE3NDRkMjQxNTE2NzllZTE5YmI0NjM0MjM3YTE5ZDAwODg4MTQ5NTI3YTY2MGExNTlkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Z_WVfGs61AhiI2ETwORGjRDv9XtJJw0_DRIBsSE-wjM)

I'd like to drop in here and address this. I'm leaving this issue open until an effective resolution is reached, and would also like to clarify that the naming here was not intended to be subversive. That said, I'm primarily interested in a best course of action to address this moving forward, which imo is to work towards standardization and consensus at this point.

Formal proposal is effectively top priority at the moment, and being actively worked on - which is imo the most productive approach here.

**Comment by aspiers** [comment]
Thanks a lot @0xacme, that's great news!

**Comment by Bubskii** [comment]
Yo aspires this shit is fair game, "teacherrr you forgot to give us homework!" Wtf is confused here?

**Comment by lightclient** [comment]
@0xacme subversive or not, you're sowing a lot of confusion in the community. You will not be officially assigned 404. There are very clear rules on how numbers for EIPs and ERCs are assigned. It's always been sequential. The next number is in the 7000s.

**Comment by TimDaub** [comment]
I hope this isn‘t considered trolling because that‘s not my intention, but considering that the concept had some level of hype and adoption in the community, wouldn‘t it now also be in the interest of the Ethereum community to have it be identified officially as ERC404? Otherwise the link gets broken and that could slow down the concept‘s memetic spread. My understanding is that this number hasn‘t been taken.

**Comment by lightclient** [comment]
@TimDaub allowing this project to not follow the process every standard has before it incentivizes future EIP/ERC authors to follow the same path. Create a standard with unofficial number, popularize it, then demand the number be assigned to them.

It's a really unfortunate situation, but this sits squarely on the shoulders of this project's authors for using the ERC nomenclature without due process.
